### ISSUE TITLE
[New option] sound=off

### ISSUE THREAD

**Issue by john-peterson** [quickemu-project/quickemu#723]
I tried audio=off sound=off --no-sound etc. I haven't found the options file in the code

I started the vm and it showed no errors. It never does. No vm running. (vagrant does this difibtw it is not async but this works too). Perhaps the biggest problem was that machine/machine.log was not flushed. No idea why. So LOG WAS EMPTY

WHen I run machine/machine.sh the error is revealed. No PA for pulse audio is not running ...facepalm. ... I'm a serious engineering why would I worry about sounds

So I install AND start PA. 

But I tired all sorts of commands to get rid of all sound devices

Just a usability issue. I will look for the config file to show a COMPLETE LIST OF CONFIG ARGUMENTS 

Update

So the whole thing is a not too big shell file. Good in testing all these OSes. Good knows it's a nightmare to boot them. Or install them. Unless there is no image on vagrant I don't even bother

TLDR

Anyway the new option would logically be next to display as

```
 --audio  : Select audio backend. 'none' (default), 'pa'

### MAIN

audio=none
```

Who in their right mind install pulse audio on a development machine

**Comment by philclifford** [comment]
This should be addressed by #705 , which will accept a "none" option for a  new "--sound-card" parameter.

**Comment by flexiondotorg** [comment]
Implemented via #705
